services/storage/src/simcore_service_storage/handlers_simcore_s3.py

```diff
@@ -160,6 +159,11 @@ async def search_files(request: web.Request) -> web.Response:
     )
 
     return web.json_response(
-        {"data": [jsonable_encoder(FileMetaDataGet(**d.model_dump())) for d in data]},
+        {
+            "data": [
+                jsonable_encoder(FileMetaDataGet(**d.model_dump()), exclude_unset=True)
```

Comment:
sure about this change?

Reply:
FileMetaDataGet is used in the webserver so in theory it should recover all non optional unset values ...  but the client side in the webserver is not up-to-date so i am not sure 100%